cloudsync_gdrive.py

```diff
@@ -337,6 +337,9 @@ def events(self) -> Generator[Event, None, None]:  # pylint: disable=too-many-lo
 
                 ohash = None
                 path = self._path_oid(oid, use_cache=False)
```

Comment:
Another option would be calling `info_oid` here, grabbing the path from info and adding the `trashed` flag to info